tests/default_vars.sh

```diff
@@ -1453,3 +1454,131 @@ export CPLMODE=hafs
 export RUNTYPE=startup
 export USE_COLDSTART=false
 }
+
+export_hafs ()
+{
+export FV3=true
+export S2S=false
+export HAFS=true
+export DATM_CDEPS=false
+export DOCN_CDEPS=false
+export THRD=1
+export INPES=$INPES_dflt
+export JNPES=$JNPES_dflt
+export TASKS=$TASKS_dflt
+export TPN=$TPN_dflt
+
+export OUTPUT_GRID='regional_latlon'
+export IMO=-999999
+export JMO=-999999
+export CEN_LON=-999.99
+export CEN_LAT=-999.99
+export LON1=-999.99
+export LAT1=-999.99
+export LON2=-999.99
+export LAT2=-999.99
+export DLON=-999.99
+export DLAT=-999.99
+export STDLAT1=-999.99
+export STDLAT2=-999.99
+export NX_LC=-999999
+export NY_LC=-999999
+export DX_LC=-999.99
+export DY_LC=-999.99
+
+export OUTPUT_GRID_2='regional_latlon'
+export IMO_2=-999999
+export JMO_2=-999999
+export CEN_LON_2=-999.99
+export CEN_LAT_2=-999.99
+export LON1_2=-999.99
+export LAT1_2=-999.99
+export LON2_2=-999.99
+export LAT2_2=-999.99
+export DLON_2=-999.99
+export DLAT_2=-999.99
+export STDLAT1_2=-999.99
+export STDLAT2_2=-999.99
+export NX_LC_2=-999999
+export NY_LC_2=-999999
+export DX_LC_2=-999.99
+export DY_LC_2=-999.99
+
+export OUTPUT_GRID_3='regional_latlon'
+export IMO_3=-999999
+export JMO_3=-999999
+export CEN_LON_3=-999.99
+export CEN_LAT_3=-999.99
+export LON1_3=-999.99
+export LAT1_3=-999.99
+export LON2_3=-999.99
+export LAT2_3=-999.99
+export DLON_3=-999.99
+export DLAT_3=-999.99
+export STDLAT1_3=-999.99
+export STDLAT2_3=-999.99
+export NX_LC_3=-999999
+export NY_LC_3=-999999
+export DX_LC_3=-999.99
+export DY_LC_3=-999.99
```

Comment:
@DusanJovic-NOAA, yes, my first though is using NX/Y and DX/Y as well, which is actually also cleaner/consistent in the model_configure template. I added _LC because, NX/Y or DX/Y are very short and might be easily replaced/redefined by other sections in the ufs rt driver scripts. But, if you would prefer, I can change it back to NX/Y and DX/Y.

Thanks!

Bin

Reply:
I personally prefer just NX,NY, DX,DY.

Reply:
Changed accordingly. Thanks!